.travis.yml

```diff
@@ -23,11 +23,24 @@ install:
   - pip install .[complete]
   # Install extra requirements for running tests and building docs.
   - pip install -r requirements-dev.txt
+  - pip install sphinxcontrib-openapi
+
+
+env:
+  global:
+    # Doctr deploy key for ke-zhang-rd/amostra
```

Comment:
Is this the deploy key for `ke-zhang-rd/amostra`? It should be for `NSLS-II/NSLS-II.github.io`. Please update the comment and, if necessary, the value of the token.